Library/SubscriptionChange.cs

```diff
@@ -110,14 +108,17 @@ internal void WriteChangeSubscriptionXml(XmlTextWriter xmlWriter)
             if (CollectionMethod.Like("manual"))
```

Comment:
If we don't need this conditional anymore, i think we can just render the string directly. Something like this:

`xmlWriter.WriteElementString("collection_method", CollectionMethod);`